### ISSUE TITLE
Support PHP 7.4

### ISSUE THREAD

**Issue by bobbingwide** [bobbingwide/oik-bwtrace#81]
A number of changes are required to oik-bwtrace to support PHP 7.4

- Need to update table of end-of-life.
```
, '7.4' => '2022-11-28'
, '7.5' => 'to be determined'
```

Note: This is the Security Support until date. See https://www.php.net/supported-versions.php

**Comment by bobbingwide** [comment]
PHPUnit tests fail with PHP 7,4
```
Notice: Trying to access array offset on value of type null in 
C:\apache\htdocs\wordpress\wp-content\plugins\oik-bwtrace\tests\test--oik-bwtrace.php 
on line 302
```

**Comment by bobbingwide** [comment]
v3.0.0 has been published saying PHP 7.4 is supported, so closing this issue.
